src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -311,6 +315,19 @@ public void resetPose(RigidTransform2 pose) {
         }
     }
 
+    @Config(tabName = "Drivebase", name = "reset modules", columnIndex = 5, rowIndex = 0)
```

Comment:
We should make the subsystem consistent about using Oblog or direct NetworkTables, but I don't think that's worth delaying merging for.